dev/spark-test-image-util/docs/build-docs

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
the script seems does not have `x` permission, you can grant it by `chmod a+x <path>` and the git will record permissions correctly in UNIX-like OS.

Reply:
Okay